afdko/Technical Documentation/OpenTypeFeatureFileSpecification.html

```diff
@@ -1771,6 +1763,12 @@ <h1>OpenType&trade; Feature File Specification</h1>
   }</pre>
 <p>along with the tag 'sbit'.</p>
 <p><a name="11"></a><b>11. Document revisions</b></p>
+<p><b>v1.22 [20 July 2018]:</b></p>
+<ul>
+<li>Removed the restriction that 'DFLT' script is allowed to have only the 'dflt' language. This now matches the changes made for version 1.8.2 of the OpenType spec.</li>
+<li>Removed the restriction that that all mark-to-base statements in a lookup and mark-to-mark must reference the the same set of mark glyph classes.
```

Comment:
Typos: that×2, the×2
Also, can
> all mark-to-base statements in a lookup and mark-to-mark must reference

be rewritten as

> all mark-to-base and mark-to-mark statements in a lookup must reference

?

Reply:
Thanks for catching these.